app/models/dataset.rb

```diff
@@ -1,3 +1,5 @@
+require 'yaml'
```

Comment:
Do we definitely need this? I think this should be automatically `require`d.

Reply:
not sure, will take it out and see